modules/fx/arrow-fx/src/main/kotlin/arrow/fx/internal/CancelablePromise.kt

```diff
@@ -1,40 +1,31 @@
 package arrow.fx.internal
 
 import arrow.Kind
-import arrow.core.Either
-import arrow.core.Left
-import arrow.core.None
-import arrow.core.Option
-import arrow.core.Right
-import arrow.core.Some
+import arrow.core.*
```

Comment:
KTLint gonna complain

Reply:
Thanks! There's a lot I need to cleanup due to many tries 😅